Islandora/islandora_internet_archive_bookreader
# Move image server to core

**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2469

Related PRs:
* https://github.com/Islandora/islandora/pull/725
* https://github.com/Islandora/islandora_paged_content/pull/158
* https://github.com/Islandora/islandora_openseadragon/pull/102

# What does this Pull Request do?

Removes the Image server configuration and makes use of a centralized one in core.

# What's new?

# How should this be tested?

Pull in related PRs

Ensure you can use the IABV with both IIIF and Djatoka servers, try using IIIF as headers. Try anything else you can think of.

# Additional Notes:

* Does this change the interface, add a new feature, or otherwise change behaviours that would require updating documentation? Updated README to refer to central image server configuration form.
* Does this change add any new dependencies? NO
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? NO
* Could this change impact execution of existing code? Yes, if auto-configuration doesn't work until it is corrected.

# Interested parties
@Islandora/7-x-1-x-committers @jonathangreen
